package.json

```diff
@@ -66,6 +66,9 @@
     "fp-ts": "^2.10.5",
     "html-to-text": "^5.1.1",
     "io-ts": "^2.2.16",
+    "io-ts-types": "^0.5.16",
+    "monocle-ts": "^2.3.13",
+    "newtype-ts": "^0.3.5",
```

Comment:
Do we need those?

Reply:
no we don't since we used the new type UlidMapFromString instead of JsonFromString. I will remove them